### DISCUSSION TITLE
What is auto-select mode?

### DISCUSSION THREAD

**Discussion by farzin-az** [davatorium/rofi#1688] [Q&A]
Hello,
I read the docs about this option, it says:

> When one entry is left, automatically select it.

I thought that this means that if I hover over an entry, for example, with <down> or <tab> after a short amount of time it is going to select it, without me selecting it explicitly, for example, with <return>.
It seems that this is not the case. I enabled this mode and nothing noticeable happened in the functionality of application. I searched and didn't find any more information about this neither in the docs nor on the web. Could someone provide me with an explanation of what this option does?

**Comment by DaveDavenport** [comment]
It does exactly what the doc sais:


    When one entry is left, automatically select it.


If you enable this option, you filter down the list and one entry remains, its auto-selected.

edit: just tested it, and it still works.

**Reply by DaveDavenport** [reply]
To auto-select and entry after a certain timeout, see the manpage.
